args/espers.py

```diff
@@ -133,6 +146,7 @@ def options(args):
         equipable = f"Balanced Random {args.esper_equipable_balanced_random_value}"
 
     result = []
+    result.append(("Starting Espers", f"Random {args.starting_espers_min}-{args.starting_espers_max}"))
```

Comment:
![image](https://user-images.githubusercontent.com/96998881/163690902-a266e5fa-5c44-4760-bd3a-97ddf93de784.png)

The resulting string is too long for the Flags menu.

Reply:
Otherwise, looks great:

27-27:
![image](https://user-images.githubusercontent.com/96998881/163690980-022c83bc-f69c-4781-838d-d53821c32123.png)

Reply:
On the beta branch, I just removed the "Espers" from "Starting Espers" and it fits.

Reply:
I think removing "Random" as it's a bit implied